decisions/wc_offmap_decisions.txt

```diff
@@ -387,142 +500,128 @@ offmap_decisions = {
 			}
 		}
 	}
-	# request_general_offmap = {
-		# only_playable = yes
-		# button_name = offmap_boons
+	request_general_offmap = {
+		only_playable = yes
+		button_name = offmap_boons
 		
-		# ai_check_interval = 12
+		ai_check_interval = 12
```

Comment:
I would change it to...
```suggestion
		ai_check_interval = 48
```
Since the AI doesn't know when to stop spawning courtiers.